js/SwordSettingsFormHandler.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file js/SwordSettingsFormHandler.js
```

Comment:
I think you can just use an `AJAXFormHandler` without extending it, no?

Reply:
modified